### PR TITLE
docs: add help description for envs command

### PR DESCRIPTION
**Pull request by zmitchell**
## Proposed Changes

Adds help descriptions for `flox envs -h`.

## Release Notes

There's more information in `flox envs -h`.

### CODE REVIEW

**Review by zmitchell — COMMENTED** [review]
Output:
```text
Show active and available environments

Usage: flox envs [--active] [--json]

Available options:
        --active  Show only the active environments
        --json    Format output as JSON
    -h, --help    Prints help information

Run 'man flox-envs' for more details.
```

File: cli/flox/src/commands/envs.rs [hunk 1]
```diff
@@ -26,6 +26,7 @@ use crate::utils::message;
 enum Mode {
     #[bpaf(long, hide)]
     All,
+    /// Show only the active environments
```
Comment (mkenigs) [inline cli/flox/src/commands/envs.rs:29]: nit
```suggestion
    /// Show only active environments
```

**Review by mkenigs — APPROVED** [review]

**Comment by mkenigs** [comment]
Is the man page still missing?

File: cli/flox/src/commands/envs.rs (hunk 1 above)
Comment (zmitchell) [reply cli/flox/src/commands/envs.rs:29]: I like my wording better

**Comment by zmitchell** [comment]
We already have a man page for `flox envs` I'm not sure why the ticket says that it's missing

**Comment by jennymahmoudi** [comment]
> We already have a man page for `flox envs` I'm not sure why the ticket says that it's missing

Building from main, when I do `man flox envs` I get this: 
image

**Comment by zmitchell** [comment]
Building doesn't install the man pages, and the man page would be `flox-envs`

**Comment by mkenigs** [comment]
> > We already have a man page for `flox envs` I'm not sure why the ticket says that it's missing
> 
> Building from main, when I do `man flox envs` I get this: image

To get man pages, you have to run an installer. And it would be `man flox-envs`

**Comment by zmitchell** [comment]
I won @mkenigs

**Review by jennymahmoudi — APPROVED** [review]

**Comment by mkenigs** [comment]
> I won @mkenigs

🏆 for @zmitchell

**Pull request merged**